devito/passes/clusters/factorization.py

```diff
@@ -76,7 +76,7 @@ def collect_const(expr):
             # -> (a + c)
             add = Add(*v)
             # -> 3.*(a + c)
-            mul = Mul(k, add, evaluate=False)
+            mul = Mul(k, add, evaluate=False) if k != 1 else add
```

Comment:
(nitpicking)

I know it used to be like that, but we're dropping the ternary operators everywhere, so the convention is now to write:

```
if k == 1:
    mul = add
else:
    mul = ...
```

sorry, conventions...